frontend/controller/actions/identity.js

```diff
@@ -960,6 +969,43 @@ export default (sbp('sbp/selectors/register', {
     // And remove transient keys, which require a user password
     sbp('chelonia/clearTransientSecretKeys', [oldIEKid, oldIPKid, IEKid, IPKid])
   },
+  'gi.actions/identity/delete': async ({
+    contractID,
+    transientSecretKeys,
+    oldKeysAnchorCid
+  }: {
+    contractID: string,
+    transientSecretKeys: Secret<string[]>,
+    oldKeysAnchorCid: string
```

Comment:
Similar request here to rename `oldKeysAnchorCid` and also add a comment below above line 991 explaining what it is and what that code is doing.

Reply:
Again, this is a CID. I think the name is pretty descriptive in this case saying exactly what it is.

Reply:
Ah ok, then just:

> add a comment below above line 991 explaining what it is and what that code is doing.